File: DayOne/Python/level_three.py
def flatten_two_dimensional_array(array):

    flattened_array = []
    for lists in array:
    
        for number in lists:
        
            flattened_array.append(number)
            
    return flattened_array
    
    
def rotate_array_by_k_positions(array, position):

    rotated_array = []

    rotated_array += (array[position+1 : ])
    rotated_array += (array[ : position+1])
            
    return rotated_array
        
            
    
def merging_and_sorting(array_one, array_two):

    merged_array = array_one + array_two
        
    for index in range(len(merged_array)-1):

        for count in range(index+1, len(merged_array)):
    
            if merged_array[index] > merged_array[count]:
        
                temporary_holder = merged_array[index]
                merged_array[index] = merged_array[count]
                merged_array[count] = temporary_holder
                
    return merged_array

File: DayOne/Java/levelThree.java
public class levelThree{

    public static int[] flattenTwoDimensionalArray(int[][] numberArray){
    
        int counter = 0;
        for (int count = 0; count < numberArray.length; count++){
           
                counter += numberArray[count].length;
                
            
        }
        
        int[] flattenedArray = new int[counter]; 
          
        int counting = 0 ;
        for (int count = 0; count < numberArray.length; count++){
            
            for (int index  = 0; index <  numberArray[count].length; index++){
                
                flattenedArray[counting++] = numberArray[count][index];
                
            }
        }
         
        return flattenedArray;
    
    
    }
  
    
    
    public static int[] mergingAndSortingArray(int[] firstArray, int[] secondArray){
    
    int[] mergedArray = new int [firstArray.length + secondArray.length];
    
    System.arraycopy(firstArray, 0, mergedArray, 0, firstArray.length);
    System.arraycopy(secondArray, 0, mergedArray, firstArray.length, secondArray.length);
    
    for (int count = 0; count < mergedArray.length-1; count++){

        for (int index  = count+1; index <  mergedArray.length; index++){
    
            if (mergedArray[count] > mergedArray[index]){
        
                int temporaryHolder = mergedArray[count];
                mergedArray[count] = mergedArray[index];
                mergedArray[index] = temporaryHolder;
            
            }
        }
    }    
        return mergedArray;

    }

    

}   


